docs/using-az-aro.md

```diff
@@ -50,9 +59,12 @@ against a development RP running at https://localhost:8443/.
     SyntaxError: invalid syntax
     ```
 
-1. Add the ARO extension path to your `az` configuration:
+1. Verify that the ARO extension path is in your `az` configuration:
 
    ```bash
+   grep 'dev_sources' ~/.azure/config
+
+   # If not found,
```

Comment:
I try to make these things so that they can be copy/pasted without concentrating too much.
How about `grep -q dev_sources ~/.azure/config || cat >>~/.azure/config <<EOF` ?